web/v1/snowflake/sql-runner/sql/standard/00-setup/01-main/01-stored-procedures.sql

```diff
@@ -89,12 +89,18 @@ CREATE OR REPLACE PROCEDURE {{.output_schema}}.column_check(SOURCE_SCHEMA VARCHA
   var delim = '~';
   var sourceColumns = list_cols_with_type(SOURCE_SCHEMA,SOURCE_TABLE,delim).split(delim);
   var targetColumns = list_cols_with_type(TARGET_SCHEMA,TARGET_TABLE,delim).split(delim);
+  var sourceColumnLengths = list_cols_with_length(SOURCE_SCHEMA,SOURCE_TABLE,delim).split(delim);
+  var targetColumnLengths = list_cols_with_length(TARGET_SCHEMA,TARGET_TABLE,delim).split(delim);
 
-  if (targetColumns.some(notIncludedIn(sourceColumns)) === true) {
-
-      throw "ERROR: Source table is missing column(s) which exist in target table.";
+  if (targetColumns.length > sourceColumns.length) {
```

Comment:
I appreciate the previous approach would fail here if the varchar lengths mismatched but perhaps we can adapt `sourceColumns` and `targetColumns` to only include the column + data type.

Reply:
Thank you for highlighting this. The code has been revised in a later commit so this became redundant.